### PR TITLE
docs: fix some typos

### PR DESCRIPTION
**Pull request by george124816**
**What this PR does / why we need it**:

This PR fix some typos that I found in all repository

**Which issue(s) this PR fixes**:

no

**Special notes for your reviewer**:

### CODE REVIEW

**Review by ivanahuckova — APPROVED** [review]
Thanks for your contribution. LGTM.

**Comment by ivanahuckova** [comment]
@george124816 Could you please merge `main` branch as currently some unrelated tests are failing. Merging main should fix it.

**Review by gtk-grafana — APPROVED** [review]

**Comment by gtk-grafana** [comment]
Hey @george124816 some backend tests are still failing, but it doesn't appear to be related to this branch, can you try pulling in fresh changes from main one more time? 
Sorry about the holdup, hopefully we can get this merged in after that!

**Comment by gtk-grafana** [comment]
Hmm, something weird is still going on with the tests, I'll re-run the jobs tomorrow and if they're still failing I'll try to hunt down whatever is going on here.

File: pkg/services/searchV2/object/testdata/dash_raw.jsonc [hunk 1]
```diff
@@ -9,7 +9,7 @@
 //  | Type: []string                                          | Type: []string | Type: []int64 | Type: []string                   |
 //  +---------------------------------------------------------+----------------+---------------+----------------------------------+
 //  | alerting/testdata_alerts.json                           |                | 16402         | a3679c3611fc3a83b0a9418e8f4cd0eb |
-//  | all-panels.json                                         |                | 23018         | a52c6dab16ecc2d3fd0b1e1edad930d6 |
+//  | all-panels.json                                         |                | 23019         | 43e90576e39fbc52a0560474489c5671 |
```
Comment (gelicia) [inline pkg/services/searchV2/object/testdata/dash_raw.jsonc:12]: Do we know why this changed? It is the test for `github.com/grafana/grafana/pkg/services/searchV2/object` that is failing, which uses this file. Could you revert the change to this file and try pushing again to see if the test passes? @gtk-grafana

File: pkg/services/searchV2/object/testdata/dash_raw.jsonc [hunk 2]
```diff
@@ -276,9 +276,9 @@
             61778,
             10392,
             25047,
-            28885,
+            28884,
```
Comment (gtk-grafana) [inline pkg/services/searchV2/object/testdata/dash_raw.jsonc:279]: @george124816 looks like there's still some changes in here that you didn't want to commit

**Review by gtk-grafana — APPROVED** [review]

**Comment by gtk-grafana** [comment]
@george124816 All merged in! Thank you so much, and congratulations on your first contribution!

**Pull request merged**